docs/getting-started/setup.mdx

```diff
@@ -1,440 +1,130 @@
 
-    An example command killing process ID `3` could look like this: `kill -9 3`.
+   If you'd like to connect to an external database, either Postgres or MSSQL, using Docker containers, please see the [external database quickstart](/getting-started/external-database-quickstart).
 
----
-
-## Environment variables
-
-If you’re running Mage using `pip` or `conda`, your local machine’s environment variables
-are accessible within the running Mage app.
-
-If you’re running Mage using Docker, you must add the following command line flags:
-
-```bash
--e SOME_VARIABLE_NAME_1=secret_value_1 -e SOME_VARIABLE_NAME_2=secret_value_2
-```
-
-The command to run Mage using Docker could look like this:
-
-```bash
-docker run -it -p 6789:6789 -v $(pwd):/home/src \
-    -e SOME_VARIABLE_NAME_1=secret_value_1 -e SOME_VARIABLE_NAME_2=secret_value_2 \
-    mageai/mageai /app/run_app.sh mage start [project_name]
-```
-
-See [here](/development/variables/environment-variables) for a list of environment variables you can configure.
+   </Accordion>
+   <Accordion title="pip/conda" icon="python">
 
-### Setting prefix for Mage url
+      1. Install Mage
 
-In some cases, you may need to add a prefix to the Mage url. You can configure the Mage server to start
-at a prefix to the url by setting the `MAGE_BASE_PATH` environment variable.
+         <CodeGroup>
+            ```bash pip
+            pip install mage-ai
+            ```
 
-Make sure to set the base path without the leading slash. For example, if you want to start Mage at
-`http://localhost:6789/base/url/path`, you would configure your environment
-variable like this:
+            ```bash conda
+            conda install -c conda-forge mage-ai
+            ```
+         </CodeGroup>
+         <sub>If you're installing on Linux and need help, check out this [community provided video walkthrough](https://www.youtube.com/watch?v=WVH7LHqND1I).</sub>
 
-`export MAGE_BASE_PATH=base/url/path`
+      2. Create new project and launch tool
 
-You can also configure the base path separately for client requests and backend routes by setting
-the `MAGE_REQUESTS_BASE_PATH` and `MAGE_ROUTES_BASE_PATH` environment variables respectively.
+         ```bash
+         mage start [project_name]
+         ```
 
-|   |   |
-| --- | --- |
-| `MAGE_BASE_PATH` | Set base path for the entire app. |
-| `MAGE_REQUESTS_BASE_PATH` | Set base path for frontend requests and urls, defaults to value of `MAGE_BASE_PATH` |
-| `MAGE_ROUTES_BASE_PATH` | Set base path for backend routes, defaults to value of `MAGE_BASE_PATH` |
+      3. Open http://localhost:6789 in your browser.
 
----
-
-## Using secret credential files in Docker
-
-If you have files containing secret credentials (e.g. GCP credentials JSON files),
-you’ll need to mount them as a volume so that those files appear in your running Docker container.
-
-### macOS
+   </Accordion>
+   <Accordion title="Kubernetes" icon="network-wired">
+      1. Start a Kubernetes cluster locally
```

Comment:
The number looks off on staging 
<img width="659" alt="image" src="https://github.com/mage-ai/mage-ai/assets/80284865/09e8a752-9e66-469b-9e0c-b16f78d1b8fe">

Reply:
Looks good on my end :) 

![image](https://github.com/mage-ai/mage-ai/assets/59450879/938985ad-fecc-449c-b7e1-1cad95621ce1)